modules/core/src/bitcoin.ts

```diff
@@ -49,7 +49,7 @@ bitcoin.HDNode.prototype.getKey = getKey;
  * @param {string} path
  * @returns {bitcoin.HDNode}
  */
-export function deriveKeyByPath(userKey: bitcoin.HDNode, path: string): bitcoin.HDNode {
+export function deriveKeyByPath(userKey: bitcoin.HDNode, path: string): bitcoin.HdNode {
```

Comment:
are both of these 2 references to hdnode correct? they have different spelling

Reply:
Good catch, will fix